app.py

```diff
@@ -34,35 +37,40 @@ def __init__(self, session_state=st.session_state):
                     st.session_state.app_initialized = True
         # Answerable and Unanswerable questions
         self.answerable_questions = {
-            "How can I contact ITS?",
-            "How can I connect to the campus Wi-Fi?",
-            "Who are the Co-Chairs for the 2024/2025 Committee?",
-            "Where are all the printers located?",
-            "What are the CoyoteLabs virtual computer labs?",
-            "Is Adobe Creative Cloud available as student software?",
-            "Does CSUSB have accessible technology?",
-            "How do I enable multi-factor authentication?",
-            "What are Coyote OneCard benefits?",
-            "Why can't I get access for wireless prints through my phone?",
+            "How can I contact ITS",
+            "How can I connect to the campus WiFi",
+            "What are the available free software for a student",
+            "Where are all the printers located",
+            "What are the CoyoteLabs virtual computer labs",
+            "Is Adobe Creative Cloud available as student software",
+            "What is information security awareness",
+            "How do I enable multi-factor authentication",
+            "What are Coyote OneCard benefits",
+            "What if i lost my campus laptop charger",
         }
         self.answerable_questions = {q.lower() for q in self.answerable_questions}
         self.unanswerable_questions = {
-            "How do I connect to Starbucks Wi-Fi?",
-            "What is a smart contract?",
-            "Can you write code for a basic Python script?",
-            "Who is the dean of CSUSB?",
-            "What class does Dr. Alzahrani teach?",
-            "Who is Hironori Washizaki?",
-            "When was CSUSB built?",
-            "What is the future impact of AI on software quality standards?",
-            "What is regression testing?",
-            "Can a student apply for a part-time job in IT support? If so, what is the process?",
+            "What are the campus gym timings",
+            "What is a smart contract",
+            "Can you write code for a basic Python script",
+            "What is the CGI phone number or email",
+            "What class does Dr. Alzahrani teach",
+            "Who is Hironori Washizaki",
+            "How can I make a payment for the tuition fee",
+            "What is the future impact of AI on software quality standards",
+            "What is regression testing",
+            "How much does parking cost for one semester",
         }
         self.unanswerable_questions = {q.lower() for q in self.unanswerable_questions}
```

Comment:
Can you change this to use the `remove_special_characters` instead of `.lower()`?